src/GrumPHP/Task/TwigLint.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Twig lint task.
+ */
+class TwigLint extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'twiglint';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(array(
+            'directory' => '.',
+            'exclude' => array('vendor'),
+            'summary' => false,
+        ));
+
+        $resolver->setAllowedTypes('directory', array('string'));
+        $resolver->setAllowedTypes('exclude', array('array'));
+        $resolver->setAllowedTypes('summary', array('bool'));
+
+        return $resolver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return $context instanceof RunContext || $context instanceof GitPreCommitContext;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $config = $this->getConfiguration();
+        $arguments = $this->processBuilder->createArgumentsForCommand('twig-lint');
```

Comment:
Does this linter handle environment specific errors like e.g. undefined functions etc?